src/application/app.test.js

```diff
@@ -11,15 +11,15 @@ describe('a doodle like app', () => {
 
   describe('create event scheduling poll', () => {
     const aTitle = 'Event title'
-    const aDate = new Date()
-    const anEventPollId = app.newEventPoll(aTitle, aDate)
+    const possibleDates = [faker.date.future(), faker.date.future()]
+    const anEventPollId = app.newEventPoll(aTitle, possibleDates)
     const anEventPoll = eventPollRepository.findById(anEventPollId)
 
     expect(anEventPoll).toBeInstanceOf(EventPoll)
 
-    it('has a title and date', () => {
+    it('has a title and possible dates for the event', () => {
       expect(anEventPoll.title).toBe(aTitle)
-      expect(anEventPoll.date).toBe(aDate)
+      expect(anEventPoll.possibleDates).toBe(possibleDates)
```

Comment:
@j-velasco do you think we should check that is an array with a lenght of more than 1 date here?

Reply:
I don't think so, as what you're testing here that the `possibleDates` of the eventPoll is the same as the one of the argument given to `app.NewEventPoll`